docs/release-notes.rst

```diff
@@ -1,6 +1,46 @@
 Release Notes
 =============
 
+Restraint 0.2.1
+---------------
+
+Upgrades:
+~~~~~~~~~
+* | Provide ease of use with Restraint commands
+  | When running commands outside of jobs on the local host,
+    some `restraint` commands require manually setting up
+    environment variables or constructing long URLs
+    before running. This can be issue if you are trying
+    to extend the watchdog in a timely fashion.
+    A new option was added which requires the
+    argument `--port <restraint-port-number>`.
+    Commands affected are rstrnt-report-log,
+    rstrnt-report-result, rstrnt-abort, and
+    rstrnt-watchdog.
+* | :bug:`1802610`: Document how to remove RHTS from Jobs
+  | Added new section :ref:`rm_rhts_guide` detailing
+    substitutes for RHTS scripts, environment variables,
+    and testinfo.desc file and associated variables.
+
+Bug Fixes:
+~~~~~~~~~~
+* | Revert process stdin behavior
+  |  Daniel to provide input.
+* | Revert logging in server_io_callback
+  |  Daniel to provide input.
+* | Return static port configuration to restraint
+  | In release 0.2.0, `restraintd` static port
+    was removed.  In this release, the static port
+    configuration is returned when `restraintd` is running
+    as a service and when spawning restraintd from the
+    restraint client.  Dynamic port configuration only
+    exists by way of the `restraint` client.
+* | :bug:`1823840`: Restraintd killed by SIGTRAP
+  | Restraintd aborted when `restraintd` service was
+    stopped.  The function chosen to log the event also
+    performed an abort.  The function was
+    replaced with one which logs with no side effects.
```

Comment:
This relates to all the changes where the calls to g_error were removed. The root cause is that errors were not handled properly because the calls to g_error introduced in 0.2.0 aborted restraintd.

1831824 is affected for sure.